book/schedule.md

```diff
@@ -1 +1,231 @@
 # Hackweek Schedule
+
+Schedule for {{ hackweek }} during {{ dates }}:

```

Comment:
Seeing this here - it looks like there are probably a bunch of other spots we can generalize to auto-fill event-specific info. I'll try and work on that some next week.

Reply:
yep, those get defined here https://github.com/uwhackweek/jupyterbook-template/blob/903ac10083e83e79b9474dc280017f4e096b1831/book/_config.yml#L9

Reply:
Can we also use this for things in the GH actions (e.g. [the DOCKER_IMAGE](https://github.com/uwhackweek/jupyterbook-template/pull/16#discussion_r718101600))?

Reply:
the jupyterbook myst stuff is all self-contained to things under the `book` subdirectory. i don't think we can run a `jb build .github/actions` script for example to do yaml substitutions. But... we could figure out consolidating to a single 'variables.yml' file for the repository with a 'rename' script. and that rename script could also modify `_config.yml` to add things under the myst_substitutions directory?... again, we can try that in a separate PR, for this one, let's just merge things in as is :)

Reply:
I did a first pass through and tried to remove all the "SnowEx" mentions (except where it's intentionally used as an example). Hopefully I didn't break more than I fixed.

Reply:
Created an issue #18 for the DOCKER_IMAGE piece of this